src/app/pages/manage-organisation/manage-organisation-registration-step-3/manage-organisation-registration-step-3.component.html

```diff
@@ -7,6 +7,21 @@ <h1 class="govuk-heading-xl page-title">
     Confirm your organisation's details
   </h1>
   <div>
+    <div class="govuk-form-group" *ngIf="isValid">
+      <div class="govuk-error-summary govuk-grid-column-three-quarters"
+          aria-labelledby="error-summary-title" role="alert" tabindex="-1"
+          data-module="govuk-error-summary">
+          <h2 class="govuk-error-summary__title" id="error-summary-title"> There is a problem</h2>
+          <div class="govuk-error-summary__body">
+              <ul class="govuk-list govuk-error-summary__list">
+                  <a onClick="return false;" (click)="setFocus()" href="javascript:;"
+                     *ngIf="isValid">
```

Comment:
Shall we change the variable name to "isInvalid"  (because, if its invalid we show the error)